src/Connectors/test/Connector.EF6Core.Test/OracleDbTypeLocatorTest.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Steeltoe.CloudFoundry.Connector.Oracle;
+using Xunit;
+
+#if NET461
```

Comment:
Should this be conditional? Only asking because it seems to be removed everywhere else and I don't know why this should only apply here.

Reply:
Yes, this package doesn't play nice outside of full framework... note this in the .csproj:
```
  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
    <!-- with netcoreapp3.1/net5.0 returns "Warning NU1701 Package restored using '.NETFramework,Version=...' instead of the project target framework. This package may not be fully compatible with your project." -->
    <PackageReference Include="Oracle.ManagedDataAccess.EntityFramework" Version="19.7.0" />
  </ItemGroup>
```